services/distribution/src/main/resources/application.yaml

```diff
@@ -95,6 +95,23 @@ services:
       min-ios: ${MIN_IOS_VERSION:0.5.0}
       latest-android: ${LATEST_IOS_VERSION:1.0.4}
       min-android: ${MIN_ANDROID_VERSION:1.0.4}
+    app-config-parameters:
+      ios-key-download-parameters:
+        cached-day-packages-to-update-on-etag-mismatch: ${KEY_DOWNLOAD_INVALIDATED_DAY_PACKAGES:"[]"}
```

Comment:
Do we need the quotes in "[]" or can they be omitted, as it is the case in the test package's application.yaml?

Reply:
They must be omitted because in the end the resulting string will contain the escaped quotes: i.e. "\"[]\""

Reply:
Perfect :)